tokio/tests/coop_budget.rs

```diff
@@ -76,3 +77,18 @@ async fn coop_budget_udp_send_recv() {
 
     assert_eq!(N_ITERATIONS / (BUDGET / 2), tracker.load(Ordering::SeqCst));
 }
+
+#[tokio::test]
+async fn test_has_budget_remaining() {
+    const BUDGET: usize = 128;
```

Comment:
Let's avoid duplicating this constant in every test. Could you move it to the top of this file?